### PR TITLE
Add `matrix` exercise

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

**Review by glennj — APPROVED** [review]
Very elegant.

**Comment by ErikSchierboom** [comment]
Thanks!

File: exercises/practice/matrix/.meta/example.8th [hunk 1]
```diff
@@ -0,0 +1,11 @@
+: >digits \ s -- a
+  "\n" s:/ ( " " s:/ ' >n a:map ) a:map
+;
+
+: row \ s n -- a
+  n:1- >r >digits r> a:_@
+;
+
+: column \ s n -- a
+  n:1- >r >digits ( r@ a:_@ ) a:map
```
Comment (glennj) [inline exercises/practice/matrix/.meta/example.8th:10]:
```suggestion
  n:1- >r >digits ( r@ a:_@ ) a:map rdrop
```
Clean up the r-stack.

**Review by axtens — CHANGES_REQUESTED** [review]
After running `column` is the rstack empty?

**Pull request merged**